2006-08-19  Robert Millan  <rmh@aybabtu.com>

        * util/i386/pc/grub-install.in: Skip menu.lst when removing
        /boot/grub/*.lst.

Index: util/i386/pc/grub-install.in
===================================================================
RCS file: /sources/grub/grub2/util/i386/pc/grub-install.in,v
retrieving revision 1.8
diff -u -r1.8 grub-install.in
--- util/i386/pc/grub-install.in	28 May 2006 23:01:43 -0000	1.8
+++ util/i386/pc/grub-install.in	19 Aug 2006 11:02:11 -0000
@@ -198,7 +198,7 @@
 
 # Copy the GRUB images to the GRUB directory.
 for file in ${grubdir}/*.mod ${grubdir}/*.lst ${grubdir}/*.img; do
-    if test -f $file; then
+    if test -f $file && [ "`basename $file`" != menu.lst ]; then
 	rm -f $file || exit 1
     fi
 done
@@ -206,7 +206,7 @@
     cp -f $file ${grubdir} || exit 1
 done
 
-# Create the core image. First, auto-detect the filesystme module.
+# Create the core image. First, auto-detect the filesystem module.
 fs_module=`$grub_probefs --device-map=${device_map} ${grubdir}`
 if test "x$fs_module" = x -a "x$modules" = x; then
     echo "Auto-detection of a filesystem module failed." 1>&2
